### PR TITLE
Change logo

### PR DESCRIPTION
**Pull request by victory-nwawunze**
I make the Logo reusable

### CODE REVIEW

File: common/Logo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import Image from 'next/image';
+import React from 'react';
+
+const Logo = () => {
+  return <Image src='/assets/logo.svg' alt='logo' layout='fill' className='img_contain' />;
```
Comment (onyegood) [inline common/Logo/index.tsx:5]: Something is wrong with your code formatting, please fix that.

**Comment by ayotidapo** [comment]
> I make the Logo reusable

Have you tested this locally

**Comment by onyegood** [comment]
sbjsbjsjsbjsjbs

File: common/Logo/index.tsx (hunk 1 above)
Comment (onyegood) [reply common/Logo/index.tsx:5]: hello

**Review by ayotidapo — CHANGES_REQUESTED** [review]

File: public/assets/logo.svg
Comment (ayotidapo) [inline public/assets/logo.svg:1]: we now have a svg, right?

**Review by onyegood — APPROVED** [review]

**Pull request merged**